src/modeler/AutoRest.Swagger/Validation/DescriptionAndTitleMissing.cs

```diff
@@ -5,24 +5,33 @@
 using AutoRest.Core.Logging;
 using AutoRest.Swagger.Validation.Core;
 using AutoRest.Swagger.Model;
+using AutoRest.Core.Properties;
 
 namespace AutoRest.Swagger.Validation
 {
-    public class DescriptionMissing : DescriptionRequired<Dictionary<string, Schema>>
+    public class DescriptionAndTitleMissing : DescriptionRequired<Dictionary<string, Schema>>
```

Comment:
The Only other rule is Descriptive Description. Do you mean that? If not, Please clarify what do you mean by extending?